dev/sbt-plugin/src/sbt-test/sbt-plugin/distribution/build.sbt

```diff
@@ -53,7 +53,8 @@ InputKey[Unit]("absence") := {
 }
 
 lazy val checkDevRuntimeClasspath = inputKey[Unit]("Checks presence of a jar in the DevRuntime classpath")
-checkDevRuntimeClasspath <<= checkDevClasspathTask(Internal.Configs.DevRuntime)
+
+checkDevRuntimeClasspath := checkDevClasspathTask(Internal.Configs.DevRuntime)
```

Comment:
I think this is missing a `.value`. This might be what is causing the test failure.